guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,290 @@
+   mobile = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/using-ecashjpg.png"
+   alt-text = "An illustration showing distorted physical paper cash along with the words USING ECASH."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+### Creating and redeeming ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "An illustration of the process of creating and redeeming ecash tokens"
+   caption = "Creating and redeeming ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+The process of creating bitcoin backed ecash is known as **minting.** The process of redeeming ecash into sats via the Lightning Network is known as **melting.** This process typically includes the following steps:
+
+#### Minting ecash
+
+**1. User Deposits Funds** - The user deposits bitcoin into a mint. Typically this is done by generating a Lightning invoice through the mint and paying it.
+
+**2. Blinded Token Creation** - Upon successful payment, the mint generates blinded tokens, which are IOUs of the deposited funds. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
+
+**3. Mint provides Tokens to User** - The user receives a set of secrets (a cryptographic value that represents ownership of a specific amount of bitcoin) that correspond to specific amounts of sats. These secrets can be combined to create an ecash token of any amount denominated in bitcoin.
+
+#### Melting ecash
+
+**4. User Initiates Melting** - The user requests to redeem their ecash tokens for bitcoin, usually this is done through the Lightning Network.
+
+**5. Token destruction** - The mint verifies and destroys the ecash tokens, ensuring they cannot be reused. This step confirms the validity of the tokens and prevents double spending.
+
+**6. Sats Transfer via Lightning Network** - The user receives an equivalent amount of bitcoin, completing the melting process and converting ecash back into bitcoin.
+
+### Sending and receiving ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
```

Comment:
In the Nuts, Bob is the mint.  Alice is the ecash sender.  Carol is who she sends the ecash to.  Should the pics here be consistent with Nuts?

Examples of Nuts with Bobmint, Alice, Carol: https://github.com/cashubtc/nuts/blob/2276462075573e26f223e8c8d575bc68d9188b39/01.md and here: https://github.com/cashubtc/nuts/blob/2276462075573e26f223e8c8d575bc68d9188b39/03.md

Reply:
Nice. I've update the illustration to show Alice and Carol. I skipped bob and just let it labeled as "ecash mint" though.